docs/node/sdk-structure.md

```diff
@@ -0,0 +1,12 @@
+# SDK Structure
```

Comment:
Again, this seems useful in a way - but as a stand alone page with out context I am not a fan.
We need to build a story for folks to follow here.